bench/wtperf/wtperf.c

```diff
@@ -559,8 +559,6 @@ worker(void *arg)
 					trk = &thread->truncate;
 				else
 					trk = &thread->truncate_sleep;
-				/* Pause between truncate attempts */
-				(void)usleep(1000);
```

Comment:
Probably, yes.

Given a choice between causing busy-wait and letting certain workloads with truncate overflow, i think I will add the pause back.